clarifai/models/model_serving/model_config/triton/triton_config.py

```diff
@@ -154,18 +155,27 @@ class TritonModelConfig:
 
   def __setattr__(self, __name: str, __value: Any) -> None:
     if __name == "image_shape":
-      self._check_and_assign_image_shape_value(__value)
+      __value = self._check_and_assign_image_shape_value(__value)
 
     super().__setattr__(__name, __value)
 
   def _check_and_assign_image_shape_value(self, value):
-    if "image" in [each.name for each in self.input]:
-      if len(value) != 2:
-        raise ValueError(
-            f"image_shape takes 2 values, Height and Width. Got {len(value)} values instead.")
-      if value[0] > MAX_HW_DIM or value[1] > MAX_HW_DIM:
-        raise ValueError(
-            f"H and W each have a maximum value of {MAX_HW_DIM}. Got H: {value[0]}, W: {value[1]}")
-      image_dims = deepcopy(value)
-      image_dims.append(3)  # add channel dim
-      self.input[0].dims = image_dims
+    _has_image = False
+    for each in self.input:
+      if "image" in each.name:
```

Comment:
> disclaimer: I'm Sumit, looking forward to contribute to Clarifai's open source, get involved with the team for also plausible employment in Tech at Clarifai, kindly excuse my 2 cents and contributions hither and thither (=:

qq- do you think, that, an Enum, instead of raw strings like "image" is a better choice? also, can we have any false positive matches as we only grep "image" in each.name? should `each.name` be explicitly "type-casted"/asserted as a str? is it better to write it as say, 
```
str(each.name).startswith(CVEnums.image.value)
```

Reply:
Thanks for the suggestion. Updated with constant variable